Cargo.toml

```diff
@@ -8,7 +8,12 @@ categories = [ "algorithms", "science" ]
 license = "MIT/Apache-2.0"
 repository = "https://github.com/rust-num/num-traits"
 name = "num-traits"
-version = "0.1.42"
+version = "0.1.43"
 readme = "README.md"
 
-[dependencies]
+[lib]
+doctest = false # multiple rlib candidates for `num_traits` found
+
+[dependencies.num-traits]
+git = "https://github.com/cuviper/num-traits"
+branch = "no_std"
```

Comment:
This is the git and branch for #30.  After that's merged and published, this dependency should become `version = "0.2"`.